Produce consume tutorial in scala (#459)
confluentinc/kafka-tutorials: _includes/tutorials/produce-consume-lang/scala/markup/dev/4_3-sbt-execution.adoc

Run the following command to create the topic and then the schemas:

+++++
<pre class="snippet"><code class="bash">{%
include_raw tutorials/produce-consume-lang/scala/code/tutorial-steps/dev/run-helpers.sh
%}</code></pre>
+++++

> Check the `build.sbt` again. You will find the tasks declared as `TaskKey[Unit]` with a main class corresponding to the
> two last tasks.

Reviewer: ```suggestion
These sbt tasks are defined in the `build.sbt` file and are declared as `TaskKey[Unit]` with the corresponding main class .
```
